geopandas/geoseries.py

```diff
@@ -74,6 +68,17 @@ def _geoseries_expanddim(data=None, *args, **kwargs):
     return df
 
 
+def _geoseries_expanddim(data=None, *args, **kwargs):
+    # pd.Series._constructor_expanddim == pd.DataFrame
```

Comment:
```suggestion
```

Reply:
I think the comment is fine to keep to understand what logic `_geoseries_expanddim` is supposed to emulate

Reply:
I'll just add a second sentence to give some context to it.